src/commands/generate/types.ts

```diff
@@ -230,6 +244,24 @@ export const ${constantName} = '${hashedKey}' as const`
     private encryptKey(variable: Variable) {
         return `dvc_obfs_${createHash('sha256').update(variable._id).digest('hex')}`
     }
+
+    private updateAliasesInRepo() {
+        if (!this.repoConfig) {
+            if (this.updateAliases) {
+                this.writer.failureMessage(
+                    'Repo configuration not found, aliases will not be updated',
+                )
+            }
+            return
+        }
```

Comment:
this could be simplified to write the error any time the repo config doesn't exist, since we're assuming `updateAliases` is true